src/actions/AuthActions.js

```diff
@@ -0,0 +1,107 @@
+import { API } from '../config/static'
+import history from '../config/hisroty'
+
+const axios = require('axios')
+
+export const loginUser = credentails => {
+  return dispatch => {
+    dispatch({ type: 'LOADING_SPINNER' })
+    dispatch({ type: 'CLEAR_ERROR_MESSAGE' })
+    axios
+      .post(`${API}/accounts/login`, credentails)
+      .then(function (response) {
+        console.log('login user response: ', response)
+        dispatch({ type: 'UPDATE_AUTH', payload: response.data })
+        localStorage.setItem('auth', JSON.stringify(response.data))
+        dispatch(fetchProfile(response.data.access))
+        history.push('/igconnect')
+      })
+      .catch(function (error) {
```

Comment:
toast on this... Make a generic api calling function which catches the API errors and shows them in a toast